pkg/scalers/gcp_stackdriver_scaler.go

```diff
@@ -0,0 +1,189 @@
+package scalers
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"k8s.io/api/autoscaling/v2beta2"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/metrics/pkg/apis/external_metrics"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+	kedautil "github.com/kedacore/keda/v2/pkg/util"
+)
+
+const (
+	defaultStackdriverTargetValue = 5
+)
+
+type stackdriverScaler struct {
+	client   *StackDriverClient
+	metadata *stackdriverMetadata
+}
+
+type stackdriverMetadata struct {
+	projectID   string
+	filter      string
+	targetValue int
```

Comment:
Could you please convert this to int64? So it is aligned with https://github.com/kedacore/keda/issues/2790. Thanks!